tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.ElevatedServer/Timer.h

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+#include "pch.h"
+
+#include <algorithm>
+#include <atomic>
+#include <chrono>
+#include <functional>
+#include <future>
+#include <mutex>
+#include <optional>
+
+using CallbackFunction = void (*)();
+
+class Timer

```

Comment:
could we have used a periodic timer instead of this timer class? https://learn.microsoft.com/en-us/uwp/api/windows.system.threading.threadpooltimer.createperiodictimer?view=winrt-22621